config/foreman-proxy-content.migrations/210112194603-enable-pulp3-content-proxy.rb

```diff
@@ -0,0 +1,7 @@
+answers.delete('foreman_proxy::plugin::pulp') if answers.key?('foreman_proxy::plugin::pulp')
```

Comment:
You can call delete safely even if it doesn't exist so this is safe:
```suggestion
answers.delete('foreman_proxy::plugin::pulp')
```